
# [CVE-2014-0472](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-0472)

## Description

- `The django.core.urlresolvers.reverse function in Django before 1.4.11, 1.5.x before 1.5.6, 1.6.x before 1.6.3, and 1.7.x before 1.7 beta 2 allows remote attackers to import and execute arbitrary Python modules by leveraging a view that constructs URLs using user input and a "dotted Python path."`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 5.1

## Scores

- **Exploitability Score**:
  - 4.9
- **Impact Score**:
  - 6.4
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2014-04-23 15:55:02
- **Vulnerability Status**:
  - Modified

## References

- **security@debian.org**: http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html
- **security@debian.org**: http://rhn.redhat.com/errata/RHSA-2014-0456.html
- **security@debian.org**: http://rhn.redhat.com/errata/RHSA-2014-0457.html
- **security@debian.org**: http://secunia.com/advisories/61281
- **security@debian.org**: http://www.debian.org/security/2014/dsa-2934
- **security@debian.org**: http://www.ubuntu.com/usn/USN-2169-1
- **security@debian.org**: https://www.djangoproject.com/weblog/2014/apr/21/security/
